give.php

```diff
@@ -244,7 +244,7 @@ private function bindClasses() {
 	private function setup_constants() {
 		// Plugin version.
 		if ( ! defined( 'GIVE_VERSION' ) ) {
-			define( 'GIVE_VERSION', '2.7.5' );
+			define( 'GIVE_VERSION', '2.8.0' );
```

Comment:
Please change this to 2.9.0

Reply:
fixed in https://github.com/impress-org/givewp/compare/dec997d3b90a...613679045944